ERCS/erc-7920.md

```diff
@@ -0,0 +1,358 @@
+---
+eip: 7920
+title: Composite EIP-712 Signatures
+description: Composite EIP-712 signatures using merkle trees.
```

Comment:
Your description is mostly a restatement of your title. You should use this space to expand on the ideas introduced in your title. In this case, perhaps a definition of "composite" would be appropriate?